bcda/client/bluebutton_test.go

```diff
@@ -236,16 +236,16 @@ func (s *BBRequestTestSuite) TestGetMetadata_500() {
 }
 
 // Sample values from https://confluence.cms.gov/pages/viewpage.action?spaceKey=BB&title=Getting+Started+with+Blue+Button+2.0%27s+Backend#space-menu-link-content
-func (s *BBTestSuite) TestHashHICN() {
+func (s *BBTestSuite) TestHashIdentifier() {
```

Comment:
We should have a unit test for `mbi-hash` _and_ `hicn-hash` since we'll be using both mechanisms in our code.

Reply:
agree but i don't know about adding a unit test for this particular method for that since the HashIdentifier function itself only takes in a string. It doesn't care if its a hicn or mbi, it's just a string value. I could be misunderstanding

Reply:
Cool, I think I was just missing where we were unit testing lookup by `mbi-hash`.  If that doesn't exist, we definitely need to add one.

Reply:
yep, we can do it on the TestGetBlueButtonID method but more so on the test client itself TestGetPatientByIdentifierHash function but we'll need the synthetic data load from bfd for the latter. U were right, this is gonna have to sit until we're ready for that data.

Reply:
@tbellj All of these unit tests appear to use the mock BlueButtonClient in `testUtils`.  No dependency on BFD is necessary in the unit tests.  I think we should pretty much duplicate all of these tests, but call them with an MBI so we are sure that we have that coverage.  Maybe an approach to take is to set the `PATIENT_IDENTIFIER_MODE` before each test to identify which mode we are testing in.

Reply:
Also, all the synthetic MBI data is now loaded into our local env if you sync this branch with `master`